src/frontend/App.tsx

```diff
@@ -71,66 +78,64 @@ const AppRoutes: React.FC<{ innloggetSaksbehandler: ISaksbehandler }> = ({
 }) => {
     const { autentisert } = useApp();
 
-    return (
-        <BrowserRouter>
-            {autentisert ? (
-                <AppInnhold innloggetSaksbehandler={innloggetSaksbehandler} />
-            ) : (
-                <ModalWrapper
-                    tittel={'Ugyldig sesjon'}
-                    visModal={true}
-                    ariaLabel={'Sesjonen har utløpt. Prøv å last inn siden på nytt.'}
+    const router = createBrowserRouter(
```

Comment:
For å støtte `useBlocker` som brukes i Prompt-avhengigheten må vi bruker denne typen initiering av ruteren...